pom.xml

```diff
@@ -287,11 +284,31 @@
                                     </rules>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>transform-tests</id>
+                                <phase>process-sources</phase>
+                                <goals>
+                                    <goal>transform</goal>
+                                </goals>
+                                <configuration>
+                                    <transformDirectory>src/test</transformDirectory>
+                                    <rules>
+                                        <jakartaDefaults>true</jakartaDefaults>
+                                        <invert>true</invert>
+                                    </rules>
+                                </configuration>
+                            </execution>
                         </executions>
                     </plugin>
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>keycloak-20</id>
+            <properties>
+                <keycloak.version>20.0.3</keycloak.version>
```

Comment:
I just noticed that in some other parts of the pom file version 20.0.4 is referenced. Is this important?